tests/test_templates.py

```diff
@@ -115,4 +115,4 @@ def test_env(env, monkeypatch):
 
                         with open(job.fn(f"script_{op}.sh")) as file:
                             reference.extend([msg] + file.read().splitlines())
-            assert "\n".join(reference) == "\n".join(generated)
+            assert "\n".join(generated) == "\n".join(reference)
```

Comment:
I swapped the direction of the template tests so that the "reference" data is treated as the base for the diff that shows up in the error output. This confuses me _every_ time I deal with these tests.